locations/templates/quarkus-application/template.yaml

```diff
@@ -196,24 +211,26 @@ spec:
           title: Namespace
           type: string
           description: The openshift/kubernetes namespace for deploying resources
-          default: iocanel
+          default: dummy
 
         imageRepository:
            title: Select a registry
            type: string
-           default: quay.io
+           default: gitea.cnoe.localtest.me:8443/giteadmin
```

Comment:
Same

Reply:
When the job will work as github workflow, I will clean the template of this PR. Please wait

Reply:
ImageRepository reverted too to Quay.io
```
        imageRepository:
           title: Select a registry
           type: string
           default: quay.io
           enum:
             - quay.io
             - gitea.cnoe.localtest.me:8443/giteadmin
             - image-registry.openshift-image-registry.svc:5000
           enumNames:
             - Quay
             - Gitea
             - Internal OpenShift Registry
```